README.md

```diff
@@ -101,6 +101,13 @@ It is very important, therefore, that needles have the correct tags. Any needle
 
 Note that tags of the form `ENV-INSTLANG-(anything)` are useless artefacts and should be removed.
 
+## Contribution Approval Guidelines
+
+Use the following guidelines to understand the level of approval needed to merge your contributions:
+1. Pull requests and merge requests to a non-default branch will require 1 visual inspection approval to merge, provided all comments are addressed and requests for followup are outstanding.
```

Comment:
Do you mean "...all comments are addressed and **no** requests for followup are outstanding."?

Reply:
Yes, yes I do. Thanks buddy.